charts/atlantis/values.yaml

```diff
@@ -11,6 +11,7 @@ orgAllowlist: <replace-me>
 # logLevel: "debug"
 
 # If using GitHub, specify like the following:
+github: {}
```

Comment:
Would it make sense to define each provider here for consistency?

Reply:
We could, but there are some other missing values as well: https://github.com/runatlantis/helm-charts/issues/230 I think I will try to fix that in a separate PR to make it more uniform.

Reply:
(I checked in this case, it shouldn't change the behaviour, but declaring the empty values for others could make it fail, so I would rather not include it here)